src/components/__tests__/RouteWrapper.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Platform } from 'react-native'
+
+import theme from '../../constants/theme'
+import render from '../../testing/render'
+import RouteWrapper from '../RouteWrapper'
+
+jest.mock('../../services/url', () => ({ openExternalUrl: jest.fn() }))
+
+describe('RouteWrapper', () => {
+  it('should have an extra container on ios', () => {
+    Platform.OS = 'ios'
```

Comment:
You should reset the platform to it's previous value at the end of the test.
sth like 
```typescript
const defaultPlatform = Platform.OS
Platform.OS = 'ios'
...
Platform.OS = defaultPlatform
```